### ISSUE TITLE
Update process is not touching flow-collector and prometheus images

### ISSUE THREAD

**Issue by fgiorgetti** [skupperproject/skupper#1129]
Update process must also udpate flow-collector and prometheus images to their latest references.

**Comment by hash-d** [comment]
@fgiorgetti, is this fixed for 1.4.0-rc3?  I have tested an upgrade from 1.3 to 1.4.0-rc3, but the flow-collector was not updated, and Prometheus was not installed.  

I thought this was addressed on #1134, but perhaps I'm mistaken?

```
$ skupper-release-binaries/1.3.0/skupper init --enable-console --enable-flow-collector                                                                                                           
Waiting 117 seconds for LoadBalancer IP or hostname...
Waiting 114 seconds for LoadBalancer IP or hostname...
Waiting 109 seconds for LoadBalancer IP or hostname...
Skupper is now installed in namespace 'manual'.  Use 'skupper status' to get more information.
$ oc get deploy -o yaml | grep -i image:                                                                                                                                                                      
          image: quay.io/skupper/skupper-router:2.3.2
        - image: quay.io/skupper/config-sync:1.3.0
          image: quay.io/skupper/service-controller:1.3.0
          image: quay.io/skupper/flow-collector:1.3.0
$ skupper version
client version                 1.4.0-rc3
transport version              quay.io/skupper/skupper-router:2.3.2 (sha256:7d0139fac2f7)
controller version             quay.io/skupper/service-controller:1.3.0 (sha256:0dbb8e4ad3e6)
config-sync version            quay.io/skupper/config-sync:1.3.0 (sha256:feb9cba58aaf)
flow-collector version         quay.io/skupper/flow-collector:1.3.0 (sha256:fb06ba2f99ba)
$ skupper update
Skupper is now updated in 'manual'.
$ oc get deploy -o yaml | grep -i image:
          image: quay.io/skupper/skupper-router:2.4.0
        - image: quay.io/skupper/config-sync:1.4.0-rc3
          image: quay.io/skupper/service-controller:1.4.0-rc3
          image: quay.io/skupper/flow-collector:1.3.0
$ skupper version
client version                 1.4.0-rc3
transport version              quay.io/skupper/skupper-router:2.3.2 (sha256:7d0139fac2f7)
controller version             quay.io/skupper/service-controller:1.3.0 (sha256:0dbb8e4ad3e6)
config-sync version            quay.io/skupper/config-sync:1.3.0 (sha256:feb9cba58aaf)
flow-collector version         quay.io/skupper/flow-collector:1.3.0 (sha256:fb06ba2f99ba)
$ skupper version
client version                 1.4.0-rc3
transport version              quay.io/skupper/skupper-router:2.4.0 (sha256:fc47e6bc44fd)
controller version             quay.io/skupper/service-controller:1.4.0-rc3 (sha256:391a1435eff8)
config-sync version            quay.io/skupper/config-sync:1.4.0-rc3 (sha256:e78b349b43ee)
flow-collector version         quay.io/skupper/flow-collector:1.3.0 (sha256:fb06ba2f99ba)

```

(the first `skupper version` still gets the old versions from the current pods, while the `oc get deploy -o yaml | grep -i image:` gets the already-changed deploy; the second `skupper version` returns the correct information, but flow-collector is still not updated.

**Comment by ajssmith** [comment]
@hash-d  The flow collector update handling is in #1134 and did not make rc3.

**Comment by hash-d** [comment]
@ajssmith, thanks; let me know if it makes it to RC4, and I'll retest.

**Comment by ajssmith** [comment]
merged #1134
